x-pack/plugins/security_solution/public/users/pages/nav_tabs.tsx

```diff
@@ -56,6 +50,12 @@ export const navTabsUsers = (
       href: getTabsOnUsersUrl(UsersTableType.alerts),
       disabled: false,
     },
+    [UsersTableType.risk]: {
+      id: UsersTableType.risk,
+      name: i18n.NAVIGATION_RISK_TITLE,
```

Comment:
<img width="1056" alt="Screen Shot 2022-05-03 at 12 24 33 PM" src="https://user-images.githubusercontent.com/6935300/166518308-31f4ee1e-ec57-44b0-9cf0-4090c0a494f0.png">
we should change this on the user details view since it is not multiple users. maybe "User risk"

Reply:
@stephmilovic 
hmm, this name doesn't make sense indeed. And we have the same problem on the Host details page. I do think that keeping the tab name consistent between pages is good. It helps me find the risk tab independent on the page I am. Maybe we should have a more generic name for this tab to keep the same name on both pages. 

Let me check with design.

Reply:
**Some context:**
The tab has the same name on the User and User details page. Should we change the tab name on the user details page? Or should we find a more generic term that could work for both pages?

@monina-n What do you think we should do here?

Reply:
@machadoum i agree with Steph's suggestions - a more accurate tab name would be "User risk." And if we're having the same naming issue with the hosts pages, it should be consistent and be "Host risk"